libpod/runtime_img.go

```diff
@@ -86,7 +86,7 @@ type ImageFilter func(*storage.Image, *types.ImageInspectInfo) bool
 // pulled. If allTags is true, all tags for the requested image will be pulled.
 // Signature validation will be performed if the Runtime has been appropriately
 // configured
-func (r *Runtime) PullImage(imgName string, allTags bool, signaturePolicyPath string, reportWriter io.Writer) error {
+func (r *Runtime) PullImage(imgName string, allTags bool, authFile string, options CopyOptions, reportWriter io.Writer) error {
```

Comment:
Should authFile be added to CopyOptions?

Reply:
Yeah, maybe that would be better. Will add it to CopyOptions

Reply:
fixed